frontend/packages/shared/src/components/GiteaHeader/VersionControlButtons/VersionControlButtons.tsx

```diff
@@ -73,6 +74,7 @@ export const VersionControlButtons = ({ org, app }: IVersionControlButtonsProps)
     });
     const { data: result } = await fetchPullData();
     if (result.repositoryStatus === 'Ok') {
+      if (onPullSuccess) onPullSuccess();
```

Comment:
Just out of curiosity; does the onPullSuccess need to be optional? 😊  Since the VersionControlButtons is only used one place I guess it can be required. But I do agree that it would be natural to set it as optional if seen in a isolated way

Reply:
Good point! Lets keep it like this to be on the safe side ;)